### ISSUE TITLE
push notifications on web return errors and don't update

### ISSUE THREAD

**Issue by NicoBerlo** [mastodon/mastodon#27709]
### Steps to reproduce the problem

new notifications that should display at the top of the notifications column don't show.
They do show once you select mentions or favorites etc and go back to all.
The console logs this error each time a new notification is pushed : 
```TypeError: t.payload.notification is undefined
    user_lists user_lists.js:163
    Immutable 4
    b Redux
    zt sounds.ts:62
    Rt errors.ts:20
    middleware loading_bar.ts:43
    Redux 2
    N notifications.js:101
    Redux 2
    onReceive streaming.js:103
    received stream.js:114
    received stream.js:113
    onmessage stream.js:244
    value index.js:14```

notifications on the same instance work via apps for example 

### Expected behaviour

new notifications should show be pushed

### Actual behaviour

they aren't

### Detailed description

_No response_

### Mastodon instance

lepoulsdumonde.com

### Mastodon version

v4.3.0alpha 6712bf8 nightly from source

### Browser name and version

firefox 119

### Operating system

win 11

### Technical details

_No response_

**Comment by 101010pl** [comment]
I confirm the error

`Uncaught TypeError: Cannot read properties of undefined (reading 'type')
    at user_lists (user_lists.js:163:40)
    at combineReducers.js:37:32
    at Array.forEach (<anonymous>)
    at combineReducers.js:36:19
    at r.Ft [as withMutations] (immutable.es.js:2361:3)
    at combineReducers.js:35:23
    at b (redux.js:288:22)
    at sounds.ts:62:14
    at errors.ts:20:12
    at loading_bar.ts:43:14
user_lists	@	user_lists.js:163
(anonymous)	@	combineReducers.js:37
(anonymous)	@	combineReducers.js:36
Ft	@	immutable.es.js:2361
(anonymous)	@	combineReducers.js:35
b	@	redux.js:288
(anonymous)	@	sounds.ts:62
(anonymous)	@	errors.ts:20
(anonymous)	@	loading_bar.ts:43
(anonymous)	@	index.js:20
dispatch	@	redux.js:691
(anonymous)	@	notifications.js:101
(anonymous)	@	index.js:16
dispatch	@	redux.js:691
onReceive	@	streaming.js:103
(anonymous)	@	stream.js:114
received	@	stream.js:113
n.onmessage	@	stream.js:244
value	@	index.js:14`

Microsoft Edge on Windows 11

**Comment by NicoBerlo** [comment]
fixed in [https://github.com/mastodon/mastodon/pull/27758](27758) !

**Comment by vmstan** [comment]
Closed by #27758
